### PR TITLE
What: Provide README in projects and link to audits

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* #292 was to create a listing of security audits, but TOC already has one.

This change addresses the need by:

* creating a readme in the projects directory to list the reviews and assessments
* create a link in the readme to the CNCF project audits

### CODE REVIEW

File: assessments/projects/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Security TAG Reviews
+
+This directory contains the artifacts of Security TAG's various assessments and
+reviews of projects. Information on the processes to by which these artifacts
+are produced can be found in [guide](/assessments/).
```
Comment (ashutosh-narkar) [inline assessments/projects/README.md:5]: Nit: link to the `guide` folder.

**Review by ashutosh-narkar — APPROVED** [review]
lgtm

**Review by lumjjb — APPROVED** [review]
LGTM - +1 on ash's comment

File: assessments/projects/README.md (hunk 1 above)
Comment (lumjjb) [inline assessments/projects/README.md:5]:
```suggestion
are produced can be found in [guide](/assessments/guide).
```

File: assessments/projects/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Security TAG Reviews
+
+This directory contains the artifacts of Security TAG's various assessments and
+reviews of projects. Information on the processes to by which these artifacts
+are produced can be found in [guide](/assessments/guide).
```
Comment (lumjjb) [inline assessments/projects/README.md:5]:
```suggestion
are produced can be found in the [guide](/assessments/guide).
```

**Pull request merged**